src/backend/router/index.ts

```diff
@@ -5,9 +5,23 @@ import { prisma } from "@/backend/utils/prisma";
 
 export const appRouter = trpc
   .router()
-  .query("get-decks", {
+  .query("get-user", {
+    async resolve() {
+      const userFromDb = await prisma.user.findFirst();
```

Comment:
Just fetching the first user--this is something I imagine we'll build on when Auth enters the picture.